docs/reference/licensing/get-trial-status.asciidoc

```diff
@@ -16,16 +16,14 @@ This API enables you to check the status of your trial license.
 [float]
 ==== Description
 
-If you want to try the features that are included in a platinum license, you can 
-start a 30-day trial. 
+If you want to try all the subscription features, you can start a 30-day trial. 
 
-NOTE: You are allowed to initiate a trial license only if your cluster has not
-already activated a trial license for the current major product version. For
-example, if you have already activated a trial for v6.0, you cannot start a new
-trial until v7.0. You can, however, contact `info@elastic.co` to request an
-extended trial license.
+NOTE: You are allowed to initiate a trial only if your cluster has not
+already activated a trial for the current major product version. For example, if
+you have already activated a trial for v6.0, you cannot start a new trial until
+v7.0. You can, however, contact `info@elastic.co` to request an extended trial.
```

Comment:
We have a specific page where customers can automatically apply for a trial extension: https://www.elastic.co/trialextension

The page has some wording that needs to be fixed, but we're trying to standardize the process to request a trial extension and it may be better to point them to this page.

What do you think?

Reply:
Great, I'll add that URL to our list of shared links and use it here, thanks!